curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-intermediate-fetch-promises-and-async-await-by-building-an-fcc-forum-leaderboard/6443ce08c89b250267c4225a.md

```diff
@@ -0,0 +1,290 @@
+---
+id: 6443ce08c89b250267c4225a
+title: Step 30
+challengeType: 0
+dashedName: step-30
+---
+
+# --description--
+
+Create a constant named `hoursAgo` that will store the number of hours that have passed since the last post. You can do this by dividing `minutesAgo` by `60`, and then rounding down to the nearest whole number. Here's an example to do that:
+
+```js
+const hoursAgo = Math.floor(minutesAgo / 60); // 1 hour = 60 minutes
+```
```

Comment:
we should not supply the full answer here

```suggestion
Create a constant named `hoursAgo` that will store the number of hours that have passed since the last post. You can do this by dividing `minutesAgo` by `60`, and then rounding down to the nearest whole number. 

```